### PR TITLE
AT_15.02_004 | <Folder>Add description

### PR DESCRIPTION
**Pull request by Kseniyaza**
https://trello.com/c/AMjoBwuE/1261-at1502004-folderadd-description

### CODE REVIEW

File: cypress/e2e/FolderAddDescription.cy.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/// <reference types="cypress"/>
+
+describe('Folder Add Descriptions', () => {
+})
+
+it("Folder>Add description", () => {
```
Comment (DariaMas7) [inline cypress/e2e/FolderAddDescription.cy.js:6]: it() should include your AT(numbers)

File: cypress/e2e/FolderAddDescription.cy.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+    cy.get('#description-link').click()
+    cy.get('textarea[class="jenkins-input   "]').type('My first test')
```
Comment (DariaMas7) [inline cypress/e2e/FolderAddDescription.cy.js:8]: My first test should be placed in fixtures

**Review by DariaMas7 — COMMENTED** [review]
Hello, it's a good practice to add your it() to an already existing file and we have one in our project
Also, I wasn't able to get your AT card since the link that you provided is broken

And I noticed some logical error in your test, US states that you are adding a description to a folder, which is a created already job, and based on your test you adding description to homepage, since all of our test starts with the home page

**Comment by KatyaKaterina33** [comment]
Please add the accepton criteria in the Test Case card and validation in the test, and the Test Case card should be contain the User Story card.

**Review by Alxndrpz — APPROVED** [review]

**Review by SawkaQA — APPROVED** [review]

**Pull request merged**